### DISCUSSION TITLE
Lombok doesn't load lombok.config

### DISCUSSION THREAD

**Discussion by adambasis** [mfussenegger/nvim-jdtls#449] [Q&A]
Hi!
In my project root, I have the lombok.config file: 
```
config.stopBubbling = true
lombok.addLombokGeneratedAnnotation = true
lombok.accessors.fluent = true
```

`fluent = true` generates `var()`, instead of `getVar()`. This does not work for me. Lombok.config does not seem to get loaded, although the default i.e. `getVar()` does work. IntelliJ has the correct behaviour, but not Neovim. I am not sure if the problem is upstream or not.
I would appreciate any help to get the lombok.config file to load.

Edit: Even when I run the CLI option delombok onto a file, it has the correct behaviour. Only when using Neovim, it doesn't.
Thank you.

**Comment by EpsilonKu** [comment]
I added `lombok.jar` to my jdtls config.
```
config.cmd = {
  'java',
  ... -- other configs
  '-javaagent:' .. home .. '/.local/bin/lombok.jar',
  ...
}
```
